### PR TITLE
[doc] Cypher-ngql CN

### PR DESCRIPTION
**Pull request by jude-zhu**
Cypher-ngql cn version

### CODE REVIEW

**Comment by darionyaphet** [comment]
Image is missed ？

File: docs/manual-CN/5.appendix/cypher-ngql.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+| 插入指定类型的点 | CREATE (:Person {age: 16}) | INSERT VERTEX <tag_name> (prop_name_list) VALUES \<vid>:(prop_value_list) |
+| 插入指定类型的边| CREATE (src)-[:LIKES]->(dst) <br/> SET rel.prop = V | INSERT EDGE <edge_type> ( <prop_name_list> ) VALUES <src_vid> -> <dst_vid>[@<ranking>]: ( <prop_value_list> ) |
+| 删除点 | MATCH (n) WHERE ID(n) = vid <br/> DETACH DELETE n | DELETE VERTEX \<vid> |
+| 删除边  | MATCH ()-[r]->() WHERE ID(r)=edgeID <br/> DELETE r | DELETE EDGE <edge_type> \<src_vid> -> \<dst_vid>[@<ranking>] |
+| 更新点属性 |SET n.name = V | UPDATE VERTEX \<vid> SET <update_columns> |
+| 查询指定点的属性| MATCH (n) <br/> WHERE ID(n) = vid  <br/> RETURN properties(n) | FETCH PROP ON <tag_name> \<vid>|
+| 查询指定边的属性  | MATCH (n)-[r]->() <br/> WHERE ID(r)=edgeID <br/> return properties(r)| FETCH PROP ON <edge_type> <src_vid> -> <dst_vid>[@<ranking>] |
+| 查询指定点的某一类关系 |MATCH (n)-[r:edge_type]->() WHERE ID(n) = vid| GO FROM \<vid> OVER  \<edge_type> |
+| 指定点的某一类反向关系 | MATCH (n)<-[r:edge_type]-() WHERE ID(n) = vid| GO FROM \<vid>  OVER \<edge_type> REVERSELY |
+| 指定点某一类关系 N 步查询  |MATCH (n)-[r:edge_type*1..N]->() <br/> WHERE ID(a) = vid <br/> return r | GO N STEPS FROM \<vid> OVER \<edge_type> |
```
Comment (whitewum) [inline docs/manual-CN/5.appendix/cypher-ngql.md]: what is N STEPS 's meaning?  1 to N or exactly N?

File: docs/manual-CN/5.appendix/cypher-ngql.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+Name                   | Cypher         | nGQL          |
+| --- | ------------ | ------------ |
+| 列出所有 labels/tags   | * MATCH (n) RETURN distinct labels(n);  <br/> * call db.labels(); | SHOW TAGS |
+| 插入指定类型的点 | CREATE (:Person {age: 16}) | INSERT VERTEX <tag_name> (prop_name_list) VALUES \<vid>:(prop_value_list) |
+| 插入指定类型的边| CREATE (src)-[:LIKES]->(dst) <br/> SET rel.prop = V | INSERT EDGE <edge_type> ( <prop_name_list> ) VALUES <src_vid> -> <dst_vid>[@<ranking>]: ( <prop_value_list> ) |
+| 删除点 | MATCH (n) WHERE ID(n) = vid <br/> DETACH DELETE n | DELETE VERTEX \<vid> |
+| 删除边  | MATCH ()-[r]->() WHERE ID(r)=edgeID <br/> DELETE r | DELETE EDGE <edge_type> \<src_vid> -> \<dst_vid>[@<ranking>] |
+| 更新点属性 |SET n.name = V | UPDATE VERTEX \<vid> SET <update_columns> |
+| 查询指定点的属性| MATCH (n) <br/> WHERE ID(n) = vid  <br/> RETURN properties(n) | FETCH PROP ON <tag_name> \<vid>|
```
Comment (whitewum) [inline docs/manual-CN/5.appendix/cypher-ngql.md]: not exact.

```
MATCH () - [r] ->()
WHERE ID(r) = $edgeID
```
or
```
MATCH (startid)-[]->(endid)
```

Comment (jude-zhu) [reply docs/manual-CN/5.appendix/cypher-ngql.md]:
```
MATCH () - [r] ->()
WHERE ID(r) = $edgeID
return properties(r)
```
返回指定边的属性，在下面这个column

**Comment by jude-zhu** [comment]
> Image is missed ？

Nope

File: docs/manual-CN/5.appendix/cypher-ngql.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
+# Comparison Between Cypher and nGQL
```
Comment (Amber1990Zhang) [inline docs/manual-CN/5.appendix/cypher-ngql.md]: Please change this title to Chinese

File: docs/manual-CN/5.appendix/cypher-ngql.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+
+## 基本概念对比
+
+|Name               | Cypher | nGQL          |
+| --- | --- | --- |
```
Comment (Amber1990Zhang) [inline docs/manual-CN/5.appendix/cypher-ngql.md]: Shall we change the first column to Chinese?

File: docs/manual-CN/5.appendix/cypher-ngql.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+    ---------------------
+
+cypher> MATCH (src:character{name:"prometheus"})-[r:father]->(dst:character)
+      > RETURN dst.name
+```
+
+- 查询百岁老人的姓名
+
+ ```
+nebula> XXX # not supported yet
```
Comment (Amber1990Zhang) [inline docs/manual-CN/5.appendix/cypher-ngql.md]: coming soon

File: docs/manual-CN/5.appendix/cypher-ngql.md (hunk 4 above)
Comment (jude-zhu) [reply docs/manual-CN/5.appendix/cypher-ngql.md]: yep

File: docs/manual-CN/5.appendix/cypher-ngql.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+| 插入指定类型的点 | CREATE (:Person {age: 16}) | INSERT VERTEX <tag_name> (prop_name_list) VALUES \<vid>:(prop_value_list) |
+| 插入指定类型的边| CREATE (src)-[:LIKES]->(dst) <br/> SET rel.prop = V | INSERT EDGE <edge_type> ( <prop_name_list> ) VALUES <src_vid> -> <dst_vid>[@<ranking>]: ( <prop_value_list> ) |
+| 删除点 | MATCH (n) WHERE ID(n) = vid <br/> DETACH DELETE n | DELETE VERTEX \<vid> |
+| 删除边  | MATCH ()-[r]->() WHERE ID(r)=edgeID <br/> DELETE r | DELETE EDGE <edge_type> \<src_vid> -> \<dst_vid>[@<ranking>] |
+| 更新点属性 |SET n.name = V | UPDATE VERTEX \<vid> SET <update_columns> |
+| 查询指定点的属性| MATCH (n) <br/> WHERE ID(n) = vid  <br/> RETURN properties(n) | FETCH PROP ON <tag_name> \<vid>|
+| 查询指定边的属性  | MATCH (n)-[r]->() <br/> WHERE ID(r)=edgeID <br/> return properties(r)| FETCH PROP ON <edge_type> <src_vid> -> <dst_vid>[@<ranking>] |
+| 查询指定点的某一类关系 |MATCH (n)-[r:edge_type]->() WHERE ID(n) = vid| GO FROM \<vid> OVER  \<edge_type> |
+| 指定点的某一类反向关系 | MATCH (n)<-[r:edge_type]-() WHERE ID(n) = vid| GO FROM \<vid>  OVER \<edge_type> REVERSELY |
+| 指定点某一类关系第 N hops 查询  |MATCH (n)-[r:edge_type*N]->() <br/> WHERE ID(a) = vid <br/> return r | GO N STEPS FROM \<vid> OVER \<edge_type> |
```
Comment (wilsonyou) [inline docs/manual-CN/5.appendix/cypher-ngql.md]: hop

File: docs/manual-EN/5.appendix/cypher-ngql.md [hunk 7]
```diff
@@ -137,7 +137,7 @@ cypher> MATCH (src:character{name:"pluto"})-[r1:lives]->()<-[r2:lives]-(dst:char
       > RETURN dst.name
```
Comment (wilsonyou) [inline docs/manual-EN/5.appendix/cypher-ngql.md]: ;

**Pull request merged**